Apsitvarkom.UnitTests/DataAccess/PollutedLocationDTOFileRepositoryTests.cs

```diff
@@ -270,5 +270,21 @@ public async Task GetByPropertyAsync_SeveralInstanceWithRequestedIdFound_FirstMa
 
         Assert.That(instance!.Radius, Is.EqualTo(2));
     }
+
+    [Test]
+    public async Task tempTest()
+    {
+        var coordinates = new Coordinates
+        {
+            Longitude = 55.7329098,
+            Latitude = 24.3557527,
+        };
+       
+        var geocoderClass = new Geocoder();
+        var result = geocoderClass.ReverseGeocode(coordinates);
```

Comment:
use `await` with async method instead of result.Result
for example here it would be
```var result = await geocoderClass.ReverseGeocode(coordinates);```